docker/Readme.md

```diff
@@ -0,0 +1,39 @@
+# Description:
+
+Files in this folder is used to make docker image of intel/kinetic:moveit_handson base environment.
+
+Dockerfile -- the base file which will be automatically loaded when run docker build command
+
+setup_docker_display.sh -- set the container to display a pop-up
+
+# Build command:
+First, clone the repo to your local system.
+```
+sudo docker build -t intel/kinetic:moveit_handson .
+```
+If proxies are needed:
+```
+sudo docker build -t intel/kinetic:moveit_handson --build-arg http_proxy=http://proxy.my.com:### --build-arg https_proxy=http://proxy.my.com:### .
+```
+# Run docker image
+After the project runs, there will be a pop-up window, you need to set the operating environment first.
+```
+./setup_docker_display.sh
+```
+Run this docker images

```

Comment:
Is it possible to distribute the docker image on-line? So that the user can download the image in case that there is no enough USB sticks for users and the user has a solid internet connection.

Reply:
You need to apply for a docker hub, and it will be affected by network bandwidth, which is not as convenient as local build.